psi4/driver/driver.py

```diff
@@ -1274,12 +1305,25 @@ def hessian(name, **kwargs):
             core.print_out("""hessian() switching to finite difference by gradients for partial Hessian calculation.\n""")
             dertype = 1
 
+    # At stationary point?
+    G0 = gradient(lowername, molecule=molecule, **kwargs)
+    nonzero_gradient = G0.rms() > 1.e-2  # pulled out of a hat
+    translations_projection_sound = not core.get_option('SCF', 'EXTERN') and \
+                                    not core.get_option('SCF', 'PERTURB_H')
+    rotations_projection_sound = translations_projection_sound and not nonzero_gradient
```

Comment:
I *really* like this `rotations_projection_sound` mechanism

Reply:
EFP should also disqualify projection, I think

Reply:
ah, EFP – good point